src/components/Home.jsx

```diff
@@ -0,0 +1,26 @@
+import React from 'react';
+
+function Home() {
+  return (
+    <>
+      <div className="homesection">
+        <h2>Welcome to our Page</h2>
+        <p className="content-paragraph">
+          Lorem ipsum dolor sit amet consectetur adipisicing elit. Veniam iure  reprehenderit.
+          Animi maxime iure dolor molestiae, aliquam blanditiis voluptatibus veritatis quae quia
+          ipsa!
+          At reiciendis.
+          Minima sint labore sed debitis sunt error iste dolor ullam blanditiis, minus nihil.
+          Voluptates at.
+          Minima sint labore sed debitis sunt error iste dolor ullam blanditiis, minus nihil.
+          Voluptates at.
+          Minima sint labore sed debitis sunt error iste dolor ullam blanditiis, minus nihil.
+          Voluptates at., tempora tempore ex.
+          Non iste obcaecati doloremque hic tenetur iusto, modi ab sunt soluta velit. , illum velit?
+        </p>
+      </div>
+    </>
+  );
```

Comment:
- [ ] Great job creating the home page, however, our design has to follow the given wireframe. Kindly adjust the `Welcome to our page `  heading to match the given layout

Your design
![image](https://github.com/misalazeem/Math-Magicians/assets/81932478/1be979cd-5629-4c04-bbcf-ee1c5abf1aa2)

Given wireframe
![image](https://github.com/misalazeem/Math-Magicians/assets/81932478/a9bc8fb8-c07a-462b-aff4-4ef560fca54e)

Kindly align the heading to the left as is in the given design